transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/CustomAttributesStore.kt

```diff
@@ -12,5 +12,5 @@ interface CustomAttributesStore {
      * Once added, they will be sent along with the next message or autostart event.
      * NOTE: in case of conflict between existing and new keys, the values of latest will be used.
      */
-    fun add(customAttributes: Map<String, String>)
+    fun add(customAttributes: Map<String, String>): Boolean
```

Comment:
Since we modify the signature of public function we must update its documentation. 
Add a line explaining what return `Boolean` means.